hermes-management/src/main/resources/application.yaml

```diff
@@ -38,6 +38,9 @@ management:
   server:
     servlet:
       context-path: /
+  health:
+    zk-health-path-prefix: /hermes/management/health
+    period: 30 # sec
```

Comment:
instead of adding the comment, the property should be called `periodSeconds` :)

Reply:
Done